modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -195,6 +195,21 @@ LayerParams ONNXImporter::getLayerParams(const opencv_onnx::NodeProto& node_prot
             lp.set("dilation_h",  saturate_cast<int32_t>(attribute_proto.ints(0)));
             lp.set("dilation_w",  saturate_cast<int32_t>(attribute_proto.ints(1)));
         }
+        else if(attribute_name == "auto_pad")
```

Comment:
The same thing as below. Conditions are duplicated.

Reply:
done